docs/quick-start-guide.md

```diff
@@ -80,43 +80,49 @@ import org.apache.spark.sql.CarbonSession._
 * Create a CarbonSession :
 
 ```
-val carbon = SparkSession.builder().config(sc.getConf)
-             .getOrCreateCarbonSession("<hdfs store path>")
+val carbon = SparkSession.builder().config(sc.getConf).getOrCreateCarbonSession("<carbon_store_path>")
 ```
-**NOTE**: By default metastore location points to `../carbon.metastore`, user can provide own metastore location to CarbonSession like `SparkSession.builder().config(sc.getConf)
-.getOrCreateCarbonSession("<hdfs store path>", "<local metastore path>")`
+**NOTE** 
+ - By default metastore location points to `../carbon.metastore`, user can provide own metastore location to CarbonSession like 
+   `SparkSession.builder().config(sc.getConf).getOrCreateCarbonSession("<carbon_store_path>", "<local metastore path>")`.
+ - Data storage location can be specified by `<carbon_store_path>`, like `/carbon/data/store`, `hdfs://localhost:9000/carbon/data/store` or `s3a://carbon/data/store`.
 
 #### Executing Queries
 
 ###### Creating a Table
 
 ```
-scala>carbon.sql("CREATE TABLE
-                    IF NOT EXISTS test_table(
-                    id string,
-                    name string,
-                    city string,
-                    age Int)
-                  STORED AS carbondata")
+carbon.sql(
```

Comment:
This is scala code format.  I think examples need not follow scala code format. 
I think the examples are wrapped to multiple lines for readability (Even if document converted to PDF)

@sraghunandan , @sgururajshetty please help to confirm the standard convention followed.

Reply:
thanks for review. guide doc should reproduce the flow simply. if not, user need to modify multi lines to make runnable, it will affect user.